kovri.md

```diff
@@ -9,76 +9,53 @@ Ring signatures obscure the sender.
 However, some personally identifiable information may be leaked at the network level when making a transaction.
-This privacy leak is addressed with technology called Kovri.
+This privacy leak is addressed with Kovri.
 
 ## 3) - The Term - Kovri
 
 Kovri is a free, decentralized, anonymity technology based on I2P's open specifications.
 Kovri uses both encryption and sophisticated routing techniques to create a private overlay-network across the Internet.
-This protected overlay allows users to hide their geographical location and IP address, essentially making Internet traffic anonymous.
+This protected overlay allows users to hide their geographical location and IP address.
 An IP address is like a home address for the Internet, so this is quite sensitive information, to say the least.
 Let's go through a few scenarios to see how this lightweight, security-focused software helps strengthen the unique privacy properties of Monero.
 
 ## 4) Examples
 
-Suppose Alice wants to send a transaction to Bob, and she beings the transaction.
+Suppose Alice wants to send a transaction to Bob, and she begins the transaction.
 In order for her transaction to have the greatest chance of being included in the next block, she needs to reach as many nodes as possible.
 Alice connects to five nodes and these nodes then forward the transaction to other nodes, which in turn rebroadcast it repeatedly until many (or all) nodes in the network have this transaction.
 Unfortunately, Alice takes a risk when broadcasting her transaction.
 Some nodes may be logging IP addresses when transaction requests are received.
 Even though wallet addresses and the amount of Monero remain private, an attacker with enough resources could attempt to associate transactions with IP addresses to determine where transactions originate from.
 This could potentially lead to an attacker knocking on Alice's door or blocking her transaction from the rest of the network.
-So what can Alice do to mitigate these threats? Well, she can use Kovri!
-If Alice sends her transaction with Kovri, nodes will no longer see her IP address, making passive surveillance tactics impractical.
 
 Now let’s imagine a different scenario.
 Suppose Charlie wants to support the Monero network by running a full node at his home.
 After a few weeks, he receives a cease and desist letter from his Internet Service Provider informing him that running a node is a violation of the terms of service.
-Charlie could have avoided this situation in the first place by anonymizing his Internet traffic with Kovri.
-Kovri substantially improves Monero's censorship resistance by making it even more difficult to detect and shut down nodes.
 
 In our last scenario, suppose Dave is an operator of a mining pool that donates a portion of block rewards to a political party or controversial cause.
 Other nodes could purposefully reject his solved blocks to express their disagreement with his political views.
```

Comment:
Since this references the "political part or controversial cause," perhaps it could end with "political or social views," or "political views or social stance."